assets/js/just-the-docs.js

```diff
@@ -53,7 +53,7 @@ function initSearch() {
       // Success!
       var data = JSON.parse(request.responseText);
 
-      lunr.tokenizer.separator = /[\s\-/]+/
+      lunr.tokenizer.separator = {{ site.search_tokenizer_separator }}
```

Comment:
If users don't set this in their `config.yml`, will the JS fail because `{{ site.search_tokenizer_separator }}` will be set to `nil`?

Reply:
In just-the-docs/_config.yml I had added:
```yaml
# Set the search token separator
search_tokenizer_separator: /[\s\-/]+/
```
I'm not familiar with how things work, but I was assuming that update would set the default, and allow site developers to see it?

Reply:
When users install this as a gem-based or remote theme, they wouldn't get this site's `_config.yml` as part of that installation. For a better experience, I try to limit the number of REQUIRED configuration in `config.yml`. 

Can we add a conditional that checks if `site.search_tokenizer_separator` is defined, and if not set it to a default manually?

Reply:
Yes, please do that. I'd have done it myself if I'd realised how it works.